ccx_messaging/downloaders/http_downloader.py

```diff
@@ -66,7 +66,7 @@ class HTTPDownloader:
     HTTP_RE = re.compile(
         r"^(?:https://[^/]+\.s3\.amazonaws\.com/[0-9a-zA-Z/\-]+|"
         r"https://s3\.[0-9a-zA-Z\-]+\.amazonaws\.com/[0-9a-zA-Z\-]+/[0-9a-zA-Z/\-]+|"
-        r"http://minio:9000/insights-upload-perma/[0-9a-zA-Z\.\-]+/[0-9a-zA-Z\-]+)\?"
+        r"[0-9a-zA-Z\-]+)\?"
```

Comment:
Is this needed or a leftover? Because looks like it can match anything with those characters

Reply:
oops, thanks!